src/StorageSync/StorageSync/help/Set-AzStorageSyncCloudEndpointPermission.md

```diff
@@ -0,0 +1,213 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://learn.microsoft.com/powershell/module/Az.storagesync/set-Azstoragesynccloudendpointpermission
+schema: 2.0.0
+---
+
+# Set-AzStorageSyncCloudEndpointPermission
+
+## SYNOPSIS
+This command sets RBAC permission required for an Azure File Sync cloud endpoint in a sync group.
+
+## SYNTAX
+
+### StringParameterSet (Default)
+```
+Set-AzStorageSyncCloudEndpointPermission [-ResourceGroupName] <String> [-StorageSyncServiceName] <String>
```

Comment:
I'm not fully understanding the Set-AzStorageSync(SEP/CEP)Permission cmdlets.

How are they intended to be invoked?
Don't these permissions get set in the Set-...Identity cmdlet?
Can the customer shoot themselves in the foot (e.g. remove RBAC to a currently syncing MI topology)?
Is there test coverage of these (I didn't see them)?

Reply:
**Don't these permissions get set in the Set-...Identity cmdlet?**
Yes, That is a migration cmdlet , a different scenario
 
**Can the customer shoot themselves in the foot (e.g. remove RBAC to a currently syncing MI topology)?**
 Yes , and that is why we are exposing permission fix operation in cmdlets and portal.
 
**Is there test coverage of these (I didn't see them)?**
Yes, Its in one of the test ps1 file. Check Test-ServerEndpointAfterIdentityMigration
 
**How are they intended to be invoked/in what circumstances?**
These permission cmdlet helps to ensure that all RBAC are correctly configured in event of 

- Accidental RBAC removal
- Move ARM operation across tenant
- etc
-